releng/tools/org.openjdk.jmc.util.releasenotes/notes/8.2/notes.xml

```diff
@@ -0,0 +1,179 @@
+				https://observablehq.com/collection/@cimi/java-mission-control.
+			</description>
+			<image>jfr/websocket.png</image>
+		</note>
+	</component>
+	<!-- ******************************* Agent ******************************* -->
+	<!-- <component> <name>JMC Agent Plug-in</name> <note version="8.1.0"> <title>New 
+		JMC Agent Plug-in</title> <description> There is now a new agent plug-in 
+		available for JMC, which allows configuring where to emit flight recording 
+		events in an already running process. </description> <image>agent/agent.png</image> 
+		</note> </component> -->
+	<!-- ******************************* Bugfixes ****************************************** -->
+	<bugfixes>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7403</bugid>
+			<synopsis>JFR parser struct types hashcode problem</synopsis>
+			<description>
+				Some JFR parser struct types were using lazily
+				initialized attributes which happen to be a part of hashCode/equals
+				computations.
+			</description>
+		</bugfix>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7532</bugid>
+			<synopsis>
+				Delays in rendering of JMX console graphs
+			</synopsis>
+			<description>
+				Sometimes the updates of the JMX console graphs would be
+				severely delayed on MacOS. This is now fixed.
+			</description>
+		</bugfix>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7068</bugid>
+			<synopsis>
+				JfrRecordingTest (uitest) hangs on the automated analysis
+				page
+			</synopsis>
+			<description>
+				Trying to run uitests on Fedora hangs on JfrRecordingTest. 
+				This was fixed after the Eclipse platform update.
+			</description>
+		</bugfix>
+	</bugfixes>
+	<!-- ******************************* Known issues ****************************************** -->
+	<knownissues>
+		<issue>
+			<area>General</area>
+			<bugid>4270</bugid>
+			<synopsis>Hibernation and time</synopsis>
+			<description>
+				After the bugfix of
+				https://bugs.openjdk.java.net/browse/JDK-6523160 in JDK 8, the
+				RuntimeMXBean#getUptime() attribute was re-implemented to mean
+				"Elapsed time of JVM process", whilst it previously was implemented
+				as time since start of the JVM process. The uptime attribute is used
+				by JMC, together with RuntimeMXBean#getStartTime(), to estimate the
+				actual server time. This means that time stamps, as well as
+				remaining time for a flight recording, can be wrong for processes on
+				machines that have been hibernated.
+			</description>
+		</issue>
+		<issue>
+			<area>JFR</area>
+			<bugid>7071</bugid>
+			<synopsis>JMC can't attach to jlinked JVMs</synopsis>
+			<description>
+				This one is still under investigation, but it seems JMC
+				can't attach to certain jlinked images.
+			</description>
+		</issue>
+		<issue>
+			<area>JFR</area>
+			<bugid>7003</bugid>
+			<synopsis>The graph and flame graph view does not work on Windows
+			</synopsis>
+			<description>
+				This is due to a bug with the Windows based browser
```

Comment:
While I guess it looks like a bug on our side, this is caused by an intentional implementation detail in how the Chromium URLs work iirc.